general_rules/ExternalComputing.tex

```diff
@@ -17,15 +13,8 @@ \section{External computing}\label{rule:robot_external_computing}
 	\item \textbf{Autonomy:} The robot has to maintain full autonomy when using \iterm{external computing resources}, 
 	  meaning there may not be a human giving the robot any kind of instructions via \iterm{external computing resources}.
 	  It is up to the team to prove to the \iaterm{Technical Committee}{TC} that there was no cheating introduced via the \iterm{external computing resources}. 
-	  For example, the use of Amazon Mechanical Turk to classify and recognize objects during a competition will be considered cheating, since effectively a human will do the classification.
```

Comment:
The tele-operation bit is still relevant. It's hard to check that but that is certainly not supposed to happen. Should be obvious to anyone in the competition though...

Reply:
Yes autonomy is still important. I only removed the example as examples are always incomplete and are redundant to the general rules.

Reply:
Just removing the specific example is fine, maybe I didn't read the diff correctly.